### PR TITLE
CREATE Portfolio project.

### PR DESCRIPTION
**Pull request by joramot**
**Build 2 section: Header and headline.**

- Create index.html
- Create style.css file
- Add the images of social media and background
- Update linters.yml file

**Defined the breakpoints to responsive website:**

- Minimum mobile screen size 375px
- Minimum desktop screen size 768px

### CODE REVIEW

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @joramot , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [x] No linter errors :white_check_mark: 
- [x] GitHub flow is used correctly :white_check_mark:  
- [x] Descriptive pull request title and summary :white_check_mark: 
- [x] Good descriptive commit messages :white_check_mark:   


## Required Changes ♻️
- [x] Please consider removing all the instructional text from your readme file, the purpose of the text is to guide you on how to fill the section in question. Once a section is filled correctly the instructional text ought to be removed. Please see the screenshot below and rectify this.

Screenshot from 2023-03-23 02-45-12



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Inter&family=Poppins&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./css/style.css">
+  <title>Jose RM Portfolio</title>
+</head>
+
+<body>
+  <section id="mobile-design">
```
[the remark this answers is not in the document]
Comment (joramot) [reply index.html:19]: Ooh ok!!, I understand, I will do the change, thanks @Felix45

File: index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      <nav class="social-icons">
+        <a href="https://github.com/joramot" rel="noopener" target="_blank"><img src="./images/github-icon.png"
+            alt="GitHub Site Link"></a>
+        <a href="https://www.linkedin.com/in/jose-rm/" rel="noopener" target="_blank"><img
+            src="./images/linkedin-icon.png" alt="LinkedIn Social Media Link"></a>
+        <a href="#"><img src="./images/angelist-icon.png" alt="Angelist Social Media Link"></a>
+        <a href="https://twitter.com/el_rulis" rel="noopener" target="_blank"><img src="./images/twitter-icon.png"
+            alt="Twitter Social Media Link"></a>
+        <a href="#"><img src="./images/medium-icon.png" alt="Medium Social MediaLink"></a>
+      </nav>
```
Comment (mckent05) [inline index.html:55]: **OPTIONAL** - For semantic reasons, having two `nav` tags on a page is not advisable. I suggest you use a `ul` tag to wrap this section and wrap each social-icon in a corresponding `li` tag. Please fix this 🙏

**Review by mckent05 — APPROVED** [review]
Hi @joramot, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (joramot) [reply index.html:55]: thanks so much @mckent05 I didn't know this, I learned new things today.

**Pull request merged**